src/core/mavsdk.cpp

```diff
@@ -98,34 +98,38 @@ void Mavsdk::register_on_timeout(const event_callback_t callback)
     _impl->register_on_timeout(callback);
 }
 
-Mavsdk::Configuration::Configuration(uint8_t system_id, uint8_t component_id) :
+Mavsdk::Configuration::Configuration(
+    uint8_t system_id, uint8_t component_id, bool always_send_heartbeats) :
```

Comment:
I'm not completely sure about having it as a state boolean. To me, that's about initiating a connection: either you passively listen until you receive heartbeats (and then start sending heartbeats), or you directly start sending heartbeats. In any case, whenever you have started sending heartbeats, that's for the duration of the connection.

Why not exposing `start_sending_hearbeat()` directly? If that's too much of an implementation detail (e.g. we don't want to mention "hearbeats"), we could call it `initiate_connection()` (or rather `initiate_connections()`, because my understanding is that it will send heartbeats on all connections)?